app/controllers/users_controller.rb

```diff
@@ -1,7 +1,7 @@
 class UsersController < ApplicationController
+
   def show
-    github_results = GithubResults.new
-    @repos = github_results.repos_limit(5)
+    @github_user = GithubUsers.new
```

Comment:
only one variable passed to the view. this is clean.

Reply:
so clean!